draft-dekater-scion-dataplane.md

```diff
@@ -1425,7 +1425,7 @@ A SCION router SHOULD accept BFD connections from its peers and SHOULD attempt t
 
 ## Link Failure Notification - SCMP {#link-down-notification}
 
-In SCION, an intermediate router cannot change the path followed by a packet, only the source endpoint can chose a different path. Therefore, to enable fast recovery, a router SHOULD signal forwarding failures to the source, via a SCMP notification (see {{I-D.dekater-scion-controlplane}} section "SCMP/Error Messages"). This allows the source endpoint to quickly switch to a different path.
+In SCION, an intermediate router cannot change the path followed by a packet, only the source endpoint can chose a different path. Therefore, to enable fast recovery, a router SHOULD signal forwarding failures to the source, via a SCMP notification (see {{I-D.dekater-scion-controlplane}} section "SCMP/Error Messages"). This allows the source endpoint to quickly switch to a different path. To that end, the source end-point needs to temporarily disqualify the broken path. Current implementations use a negative cache with entries retained for 10s.
```

Comment:
No; I disagree. We say that SCMP errors are optional, which means that disqualifying broken paths is unavoidably optional. So, there is no SHOULD. It is true that *in order to* stop using the path, the endpoint *should*. But that's lower case. Since lower-case "should" or "must" are pretty-much forbidden in RFCs.... also, even disqualify is too strong a term (one could just lower its preference score; or whatever makes more sense per the implementation).

How about: "To that end, the source endpoint needs to give lower preference to the broken path. Current implementations use..."

Reply:
How about using MAY?

Reply:
Ok, SHOULD is back. Let's not mince hair forever.